packages/extension/src/ui/features/accountTokens/tokens.state.ts

```diff
@@ -274,19 +198,32 @@ export const useFungibleTokens = (
     }
   )
 
-  return fungibleTokens || []
+  const tokenDetailsIsInitialising = !error && !fungibleTokens
```

Comment:
Do we need to add the `!allUserTokensError` to the conditions?

Reply:
Great question, I have put it [here](https://github.com/alephium/extension-wallet/blob/improve-get-balance/packages/extension/src/ui/features/accountTokens/tokens.state.ts#L206)